src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -290,15 +290,17 @@ void ExecutorWorker::handleCreateOp(SyncOpPtr syncOp, std::shared_ptr<AbstractJo
 
             // Remove from update tree
             if (!affectedUpdateTree(syncOp)->deleteNode(syncOp->affectedNode())) {
-                LOGW_SYNCPAL_WARN(_logger, L"Error in UpdateTree::deleteNode: node name="
+                LOGW_SYNCPAL_WARN(_logger, L"Error in UpdateTree::deleteNode: affectedNode name="
                                                    << SyncName2WStr(syncOp->affectedNode()->name()).c_str());
```

Comment:
Should we be more careful about dereferencing `syncOp->affectedNode()`? Or is this paranoïa ?

Reply:
Thanks. No need to change anything if there is no good reason to do so.